EIPS/eip-7702.md

```diff
@@ -57,7 +57,7 @@ At the start of executing the transaction, for each `[contract_code, y_parity, r
 2. Verify that the contract code of `signer` is empty.
 3. Set the contract code of `signer` to `contract_code`.
 
-At the end of the transaction, set the `contract_code` of each `signer` back to empty.
+At the end of the transaction, set the `contract_code` of each `signer` back to empty. Any storage set for the `signer` must also be cleared.
```

Comment:
I'd say not cleared, but prevent SSTORE in the first place.  Otherwise it might have unintended consequences to code that assumes the SSTORE was not implicitly a TSTORE.  See my Safe example in the EM thread.

Reply:
This is what I originally proposed, both in EIP-5806 and [here](https://github.com/ethereum/EIPs/pull/8527#discussion_r1593099283).

I see that #8539 is specifying that part, so I'll change this PR to be only about create/create2/selfdestruct